src/renderer/components/+helm-releases/release-details/release-details.tsx

```diff
@@ -70,7 +71,11 @@ class NonInjectedReleaseDetails extends Component<ReleaseDetailsProps & Dependen
   }
 
   @computed get details() {
-    return this.props.releaseDetails.value.get();
+    const details = this.props.releaseDetails.value.get();
+
+    assert(details);
+
+    return details;
   }
```

Comment:
Given that this is only used in the `renderNotes` function below and that is only used once in `renderContent`, I think we probably could just drop this computed and just pipe the value through.

Reply:
Resolving this because I implemented details to remove many bugs and blindspots.